src/models/User.js

```diff
@@ -1,30 +1,25 @@
 export default (sequelize, DataTypes) => {
-  const User = sequelize.define('User', {
-    first_name: {
+  const User = sequelize.define('users', {
+    firstName: {
       type: DataTypes.STRING,
-      allowNull: false
+      allowNull: true
     },
-    last_name: {
+    lastName: {
       type: DataTypes.STRING,
-      allowNull: false
+      allowNull: true
     },
     email: {
       type: DataTypes.STRING,
-      allowNull: false,
-      unique: true
+      unique: true,
+      allowNull: false
     },
     password: {
       type: DataTypes.STRING,
       allowNull: false
     },
-    is_admin: {
-      type: DataTypes.BOOLEAN,
-      defaultValue: false,
-
-    }
-  }, {});
+  });
   User.associate = (models) => {
-    // associations can be defined here
+    User.hasOne(models.Profile, { as: 'theUser', foreignKey: 'userId' });
```

Comment:
I think you need an `onDelete: 'CASCADE'  here.

Reply:
OnDelete cascade on associations @uthdev. I don't think it's possible... Its not required, also this setup is a basic configuration, as time goes by, if anybody needs to modify them, then it gest modified. This might not even be the setup we end up using.. I hope you understand.

Reply:
@funmi5, It is very possible. I think that's even the standard. It's why they call it association. That if you delete users table, Why should the corresponding user's profile be left standing? If a user decides to close it account on our app, we don't want the profile to be left standing after deleting the user.
You can check this out.
 https://www.oriechinedu.com/posts/getting-started-with-sequelize-and-postgres/
It's pretty straightforward.

Reply:
As I said, as time goes by, it can be modified, these are the basic configurations @eokwukwe requested for. I don't think anybody is thinking of delete users table as for now. `we can always modify Sequelize models`

Reply:
Ok @funmi5, We can leave that for now.